src/test/java/uk/nhs/digital/nhsconnect/nhais/service/FhirToEdifactServiceTest.java

```diff
@@ -83,44 +104,85 @@ public void when_convertedSuccessfully_dependenciesCalledCorrectly() throws Exce
     }
 
     @Test
-    public void when_convertedSuccessfully_edifactIsCorrect() throws Exception {
-        Patient patient = createPatient();
+    public void when_convertedSuccessfully_edifactIsCorrect() {
```

Comment:
Need a unit test for each acceptance type. See Appendix J for example of each type.

Do **NOT** include "Type 5 (Ex-Services). We've learned today that this is not used and we don't be supporting it.

Reply:
will fix-forward using new integration test framework